### DISCUSSION TITLE
How to migrate a Genshin private server to a normal Genshin server

### DISCUSSION THREAD

**Discussion by oivel** [Melledy/Grasscutter#2148] [Support] [answered]
I have a Genshin private server that works just fine, it's just when I try to log into the account using the normal Genshin launcher, none of the game data transferred over to the normal Genshin servers from the private servers. I haven't been able to find a solution, so maybe you have one? Im not sure if I'm able to sense it's running off of a different server.

**Comment by KingRainbow44** [comment, marked as answer]
thats not how private servers work, you have separate data.
